lib/collection/src/collection.rs

```diff
@@ -80,7 +82,7 @@ impl Collection {
     ) -> CollectionResult<UpdateResult> {
         let sndr = self.update_sender.clone();
         let (callback_sender, callback_receiver) = if wait {
-            let (tx, rx) = async_channel::bounded(1);
+            let (tx, rx) = mpsc::channel(1);
```

Comment:
Could we use a [oneshot::channel()](https://docs.rs/tokio/latest/tokio/sync/index.html#oneshot-channel) here instead?
It is supposed to be optimized for this use case.

Reply:
Done